packages/admin/admin/src/form/Field.tsx

```diff
@@ -51,7 +51,7 @@ export function Field<FieldValue = any, FieldElement extends HTMLElement = HTMLE
 }: FieldProps<FieldValue, FieldElement>) {
     const { disabled, variant, fullWidth } = otherProps;
 
-    const { mutators } = useForm();
+    const { mutators, getFieldState } = useForm();
```

Comment:
Why not use useField() like in the linked issue?

Reply:
It does not work as intended when using useField()